js/classes/entities/midground/weapon/slingshot.js

```diff
@@ -11,11 +11,14 @@ class Slingshot {
         this.isFiring = false;
         this.rotation = 0;
 
-        this.shootTimer = 0;
+        this.timeSinceLastShot = 0;
+        // this.timeSinceReload = 0;
 
         this.playedStretchSound = false;
 
         this.start = new Vector(0, 0);
+
+        // this.progressBar = new ProgressBar(new Vector(0, 0), 100, 10, "green", "red");
```

Comment:
Why are these commented out? Is this intentional?

Reply:
Good question. I was trying implement a ProgressBar class very similar to Trae's HealthBar. I was thinking the slingshot could charge up instead of firing instantly, with a progress bar displaying its current level of charge.